autolab/Autolab
# Gradebook UIUX

Addresses #2034. 
- Double click on resize to autosize and fit all column headers 
- Removed unintuitive column picker
- Hover dropdown for hide/unhide columns and force fit
- Hover name for version is now appended with assessment name to minimize confusion
- Columns that are hidden are saved in local storage and are now sticky

## How Has This Been Tested?
- Check that double clicking the border of a column header resizes the columns and expands them
<img width="514" alt="Screenshot 2024-04-22 at 1 14 07 PM" src="https://github.com/autolab/Autolab/assets/65066221/7ea110b6-a074-4aca-8a6f-65150adb0f7b">

- Hover over versions and check that full name appears
<img width="409" alt="Screenshot 2024-04-22 at 1 14 52 PM" src="https://github.com/autolab/Autolab/assets/65066221/047b63b9-80b1-43df-87a7-d6b41aed1f7f">

- Hide Columns work as expected by hiding a column
- Hide Columns are sticky even after refresh
- Unhide columns works as expected by unhiding all columns that were previously hidden and refreshes local storage

## Types of changes

## Checklist:
- [] My change requires a change to the documentation, which is located at [Autolab Docs](https://docs.autolabproject.com/)
